README.md

```diff
@@ -11,47 +11,41 @@
 
 This documentation describes the process to apply and validate an IoT Device as FIWARE Ready.
 
-To join FIWARE-ready IoT Devices Program, you need to follow a series of steps, including an simple validation of your
-product (implementation), where you will connect it to a FIWARE public instance and you will perform a series of tests:
+To be featured as a FIWARE-ready IoT Device, you need to follow some steps, including a simple validation of your
+product (implementation), where you will connect it to one of the FIWARE public instances available and you will have to
+perform the following tests:
 
 -   IoT services creation (optional)
 -   Device registration
 -   Obtain the list of devices (optional)
 -   Send measurements
 -   Read measurements (optional)
```

Comment:
I think that IoT service creation should not be optional the same for the read measurements.

Reply:
**Won't fix** - at least not with this PR - this is a separate issue for discussion. I'm just altering the intro